src/main/webapp/i18n/de/learningGoal.json

```
@@ -7,6 +7,15 @@
             "learningGoalButton": "Lernziele",
             "title": "Lernziele",
             "description": "Beschreibung",
+            "taxonomy": "Taxonomy",
```

```suggestion
            "taxonomy": "Taxonomie",
```
Is there no better German word?